.github/workflows/pre_release_version_bump.yml

```diff
@@ -0,0 +1,41 @@
+name: Pre-Release Version Bump
+
+on:
+  workflow_dispatch:
+
+jobs:
+  pre_release_version_bump:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Derive versions
+        id: versions
+        run: |
+          old_version=`grep -oP '(?<=__version__ = ").*(?=")' xir/_version.py`
+          new_version=${old_version%-dev}
+
+          echo "::set-output name=old_version::$old_version"
+          echo "::set-output name=new_version::$new_version"
```

Comment:
These are just for debugging I assume? Can probably get rid of them if we are confident it works ?

Reply:
Actually, this is the [recommended way of setting output parameters](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-output-parameter) in GitHub Actions workflow steps!